pymc3/smc/smc_utils.py

```diff
@@ -0,0 +1,301 @@
+"""
```

Comment:
Is this a rebase problem?

Reply:
Yes, I believe so -- I think that when git ran `diff` it decided, mistakenly, that I had added this whole file.

Thanks for catching that, deleting.